.gitignore

```diff
@@ -1,8 +1,10 @@
+/.idea/
```

Comment:
This should not be part of the library: please use a global `.gitignore` as described in https://docs.github.com/en/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer